core/python/ntuple_generator.py

```diff
@@ -68,7 +68,14 @@ def generate_process(year, useData=True, isDebug=False, fatjet_ptmin=120.):
     if (year=="2016v2" or year=="2016v3"):
         met_sources_GL.extend(['slMETsCHS'])
 
-
+    #try eras for correct b-tagging
+    from Configuration.StandardSequences.Eras import eras
+    if year == "2018":
+        Process = cms.Process("DUMMY", eras.Run2_2018) 
```

Comment:
these need to be `process = ...`?

Reply:
would then need to remove the `process = cms.Process...` line at L171

Reply:
hm, not sure why it worked since DUMMY process is indeed not used anywhere. Will fix it in the next commit.